packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -3824,13 +3879,23 @@ describe('StripeHelper', () => {
         sandbox.stub(stripeHelper, 'fetchCustomer').resolves(customer);
         const result = await stripeHelper.extractCustomerDefaultPaymentDetails({
           uid,
-          email,
         });
         assert.deepEqual(result, {
           lastFour: null,
           cardType: null,
+          country: null,
+          postalCode: null,
         });
       });
+
+      it('does not fetch the customer when a customer is given as the argument', async () => {
+        const customer = deepCopy(mockCustomer);
+        sandbox.stub(stripeHelper, 'fetchCustomer').rejects();
+        await stripeHelper.extractCustomerDefaultPaymentDetails({
+          customer,
+        });
+        sinon.assert.notCalled(stripeHelper.fetchCustomer);
+      });
```

Comment:
I'm hoping we can avoid this test if we change the param for `setCustomerLocation` to take in a `uid`.